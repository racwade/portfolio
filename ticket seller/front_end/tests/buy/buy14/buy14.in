login
test_admin
buy
test_sell
Many
5
yes
logout