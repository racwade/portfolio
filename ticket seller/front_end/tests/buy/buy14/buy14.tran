04 Many                test_sell      005 000100
00 test_admin          AA 000009499