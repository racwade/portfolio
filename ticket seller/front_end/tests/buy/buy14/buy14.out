welcome to the ticket seller system
enter username:
logged in as test_admin
enter seller's username:
enter event name:
enter number of tickets to buy:
total cost for 5 tickets is 500.00. confirm? (yes/no):
purchased 5 tickets to Many from test_sell for 500.00
bye